.github/fixtures/test-gitlab-integration/expected.md

```diff
@@ -0,0 +1,69 @@
+* Merge branch 'feature/tmux-improvements' into 'master' by @dark0dave in #2
+* Added in tmux first time setup, made zsh remmeber history from all tmux windows and put python install in debian setup by @dark0dave
+* Merge branch 'feature/update' into 'master' by @dark0dave in #3
+* Update README.md by @dark0dave
+* Merge branch 'readmeUpdate' into 'master' by @dark0dave in #5
+* Added basic CI file by @dark0dave
+* Merge branch 'feature/gitlabCI' into 'master' by @dark0dave in #4
+* Update README.md by @dark0dave
+* Merge branch 'readMeFix' into 'master' by @dark0dave in #6
+* Fixed bug where zshrc.local was not sourced by @dark0dave
+* Merge branch 'bugfix/installScript' into 'master' by @dark0dave in #7
+* Added nvm and fixed bug with installer by @dark0dave
+* Merge branch 'feature/nvm' into 'master' by @dark0dave in #8
+* Fixed nvm directory bug by @dark0dave
+* Merge branch 'bugfix/nvm' into 'master' by @dark0dave in #9
+* Added vim plugins by @dark0dave
+* Ignoring swap files by @dark0dave
+* Merge branch 'feature/vimPlugins' into 'master' by @dark0dave in #10
+* Updates to vim by @dark0dave
+* Merge branch 'feature/vimPluginsUpdate' into 'master' by @dark0dave in #11
+* Fixed tab issue and install bug with vundle by @dark0dave
+* Merge branch 'feature/tabFixUp' into 'master' by @dark0dave in #12
+* Fixed bug: Install had run before, fails to reinstall vim by @dark0dave
+* Merge branch 'feature/updateVimInstall' into 'master' by @dark0dave in #13
+* Updates with do not track, rust and copy pasta alias by @dark0dave
+* Merge branch 'feature/copyPastaAlias' into 'master' by @dark0dave in #14
+* Added ruby by @dark0dave
+* Merge branch 'feature/ruby' into 'master' by @dark0dave in #15
+* Fixed up ruby and added golang by @dark0dave
+* Merge branch 'feature/golang' into 'master' by @dark0dave in #16
+* Add flutter and moved to fast syntax highlighting by @dark0dave
+* Merge branch 'feature/flutter' into 'master' by @dark0dave in #17
+* Clock + Weather by @dark0dave
+* Goodbye oh my zsh by @dark0dave
+* Merge branch 'feature/zimfw' into 'master' by @dark0dave in #18
+* feat(rtx): One version manager to bring them all, and in the darkness bind them by @dark0dave
+* Merge branch 'feature/rtx' into 'master' by @dark0dave in #20
+* fix(remove): Remove home/bin from path by @dark0dave
+* feat(wezterm): Add wezterm to dotfiles and update vim by @dark0dave
+* feat(scroll): Updates for scroll bar by @dark0dave
+* feat(rtx): Use rtx for every thing, change tool-verions to .rtx.toml, add zig, update readme by @dark0dave
+* feat(pre-commit): Add pre-commit by @dark0dave
+* feat(shellcheck): Add shellcheck by @dark0dave
+* fix(node): Add node back by @dark0dave
+* feat(latest): Latest or die by @dark0dave
+* chore(readme): Bump readme by @dark0dave
+* feat(vim): Trash vim by @dark0dave
+* fix(simple): Simplify install script by @dark0dave
+* feat(update): Remove vim by @dark0dave
+
+## New Contributors
+* @dark0dave made their first contribution in #
```

Comment:
Any idea why the MR number is empty? Maybe we can avoid this by checking it in the template.

Reply:
Its because I am skipping merge commits.

Reply:
I'll add back in